lib/core/src/lib/styles/_mat-selectors.scss

```diff
@@ -142,3 +142,4 @@ $mat-form-field-error-wrapper: '.mat-mdc-form-field-error-wrapper';
 $mat-text-field-textarea: '.mdc-text-field--textarea';
 $mat-list-item-disabled: '.mdc-list-item--disabled';
 $mat-chip-focus-overlay: '.mat-mdc-chip-focus-overlay';
+$mat-mdc-form-field-hint-wrapper: '.mat-mdc-form-field-hint-wrapper';
```

Comment:
This selector is already present in the mat-selectors.sccs file at line num [101](https://github.com/Alfresco/alfresco-ng2-components/blob/949706cde4dd8ce00f8f497dfa7fc8972bc0a537/lib/core/src/lib/styles/_mat-selectors.scss#L101). Can we remove the duplicate? You would also need to update the code where this selector is used, since its named a little differently.